static/src/javascripts/projects/commercial/modules/third-party-tags.js

```diff
@@ -65,15 +64,16 @@ const insertScripts = (
         }
     });
 
-    const onCMPConsentNotification = isInTcfv2Test()
+    const onCMPConsentNotification = shouldUseSourcepointCmp()
         ? onConsentChange
         : oldCmp.onIabConsentNotification;
 
     onCMPConsentNotification(state => {
         let consentedAdvertisingServices = [];
-        if (typeof state === 'boolean') {
+        if (typeof state.ccpa === 'boolean') {
```

Comment:
This line can be simplified now. Same in any other instances.
```suggestion
        if (state.ccpa) {
```